### PR TITLE
Some author improvements

### PR DESCRIPTION
**Pull request by elbotho**
@metzgaria here you can see the changes:
https://frontend-git-author-improvements-serlo.vercel.app/
- caveats: new strings are not translated in the preview
- logged in features might not be testable

### CODE REVIEW

**Review by inyono — APPROVED** [review]
LGTM

**Comment by elbotho** [comment]
Bundle size going crazy, needs investigation.

**Comment by elbotho** [comment]
@inyono I'm a bit stuck, did you see something that could increase bundle size by `>800kb` in here?

File: src/components/comments/comment-area-all-threads.tsx [hunk 1]
```diff
@@ -1,23 +1,42 @@
+import { faWarning } from '@fortawesome/free-solid-svg-icons'
```
Comment (inyono) [inline src/components/comments/comment-area-all-threads.tsx]: This?

File: src/components/pages/unrevised-revisions.tsx [hunk 2]
```diff
@@ -1,8 +1,11 @@
+import { faEye } from '@fortawesome/free-solid-svg-icons'
```
Comment (inyono) [inline src/components/pages/unrevised-revisions.tsx]: Same here

**Comment by inyono** [comment]
Only thing I noticed are FA icons that are loaded from the package itself (that might not be tree-shakeable). Should be imported like other icons. If that isn't it, we'd have to look at it locally in detail I suppose.

**Comment by elbotho** [comment]
Okay thanks.
Those are somewhat tree-shakable and should only make a tiny difference. 
Since I spend like 5 hours yesterday tracing a few bundle size kbs I'll keep this one for next week 🙈

**Comment by inyono** [comment]
Okay, checked out locally and it seems like the editor bundle is added "everywhere".

File: src/components/head-tags.tsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { useRouter } from 'next/router'
 
 import { useInstanceData } from '@/contexts/instance-context'
 import { BreadcrumbsData, HeadData } from '@/data-types'
+import { sandboxUrlStart } from '@/fetcher/fetch-editor-data'
```
Comment (inyono) [inline src/components/head-tags.tsx]: Maybe

Comment (inyono) [reply src/components/head-tags.tsx]: Yep: https://github.com/serlo/frontend/pull/1828#issuecomment-1242559133

**Pull request merged**